cocotb/share/makefiles/simulators/Makefile.xcelium

```diff
@@ -107,8 +107,8 @@ GPI_LIB = $(COCOTB_VPI_LIB) $(COCOTB_VHPI_LIB)
 $(COCOTB_RESULTS_FILE): $(SIM_BUILD) $(HDL_SOURCES) $(CUSTOM_COMPILE_DEPS) $(CUSTOM_SIM_DEPS) $(COCOTB_LIBS) $(GPI_LIB)
 	-@rm -f $(COCOTB_RESULTS_FILE)
 
-	LD_RUN_PATH=$(PYTHON_LIBDIR):$(LD_RUN_PATH) PYTHONPATH=$(LIB_DIR):$(PWD):$(PYTHONPATH) \
-	LD_LIBRARY_PATH=$(LIB_DIR):$(LD_LIBRARY_PATH) MODULE=$(MODULE) TESTCASE=$(TESTCASE) TOPLEVEL=$(TOPLEVEL) COCOTB_SIM=1 \
+	PYTHONPATH=$(PWD):$(PYTHONPATH) \
```

Comment:
Fine as future cleanup, but... isn't including `pwd` the default anyway? Or does the pwd change due to the embedded interpreter?
```suggestion
```

Reply:
Let's leave that till that PR then.

Reply:
As I remember `PWD` is not added by default. I think it may be better to add this inside python software with `sys.path` than here.